studio/schemas/objects/sections/openness.ts

```diff
@@ -0,0 +1,42 @@
+import { defineField } from "sanity";
+
+import { titleID } from "studio/schemas/fields/text";
+export const opennessID = "opennessSection";
+
+export const opennessSection = defineField({
```

Comment:
Maybe make everything (title, image, description) required?

Reply:
Fixed in https://github.com/varianter/variant.no/pull/997/commits/005770a97b5d15b2ac207e864840680635c05d98